src/EncryptService.php

```diff
@@ -66,12 +66,12 @@ function decrypt($text) {
     $settings = \Drupal::config('encrypt.settings');
 
     // Load the key.
-    $key = $this->key_manager->getKey($settings->get('encryption_key'));
+    $key = \Drupal::getContainer()->get('key_manager')->getKey($settings->get('encryption_key'));
```

Comment:
See feedback above. May be an issue with what we're having people select in the EncryptSettingsForm